packages/@blockprotocol/graph/src/types/subgraph/edges/outward-edge-alias.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * A collection of 'aliases' which describe various variants of outward edges in more accessible-forms
+ */
+
+import { EntityIdAndTimestamp, OutwardEdge } from "./outward-edge";
```

Comment:
```suggestion
import { EntityIdAndTimestamp, OutwardEdge } from "./outward-edge.js";
```

Reply:
I'm confused, I swear I checked all of them 😭 

[Missing .js import](https://github.com/blockprotocol/blockprotocol/pull/820/commits/836126f168db91e4f2f6f46a7d46d0dae03ac11d)